### DISCUSSION TITLE
Where are preview images saved?

### DISCUSSION THREAD

**Discussion by Bansexual** [d8ahazard/sd_dreambooth_extension#206] [Q&A]
Where are preview images saved? They're not in the main outputs directory. They're not in any of the model/dreambooth/ subdirectories. It does not show the preview images in WebUI. Classifier images are saved in the classifier directory as they should be (and appear in WebUI), but preview images....🤷‍♂️

**Comment by Slowly-Grokking** [comment]
stable-diffusion-webui/models/dreambooth/MODELNAME/samples
